main.go

```diff
@@ -74,6 +77,10 @@ func Main() int {
 		}
 	}
 	log.Infof("grol %s - welcome!", cli.LongVersion)
+	memlimit := debug.SetMemoryLimit(-1)
+	if memlimit == math.MaxInt64 {
+		log.Warnf("Memory limit not set, please set GOMEMLIMIT=1GiB or similar")
```

Comment:
Maybe add something about the fact you are talking about an environment variable

Reply:
changing to
```
12:07:34.993 [WRN] Memory limit not set, please set the GOMEMLIMIT env var; e.g. GOMEMLIMIT=1GiB
```